sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -126,6 +115,16 @@ class DataFrameReader private[sql](sqlContext: SQLContext) extends Logging {
   }
 
   /**
+   * Loads input in as a [[DataFrame]], for data sources that require a path (e.g. data backed by
+   * a local or distributed file system).
+   *
+   * @since 1.4.0
+   */
+  def load(path: String): DataFrame = {
```

Comment:
this one is actually useful for both R / Python, and required for data sources that don't support paths. So I'm just removing the todo to remove it.